[API] get credential endpoint documented. Credentials url conditions mentioned. (#92)
UnumID/UnumID.github.io: docs/api-overview.mdx

    "expirationDate?": string, // when the credential expires as a milliseconds since unix epoch; optional
    "status": 'valid' | 'revoked' // an enum to denote whether the credential is valid (all credentials are valid when first issued) — note that this is independent of the expirationDate
  }],
  "url"?: string // Only present if partnerUuid was provided and conditions are met. See the Issuance Guide for more information.
}
```

> The created and issued credentials contains an `id` that you should store. You'll need this, for example, if you ever want to revoke the credential. **We recommend storing the entire credential, indexed on the credential `id`.** However, it is also an acceptable to store credential `id`s associated with a User and use the [Get Credential](/api-overview#get-credential) or [Get All User Credentials](/api-overview#get-all-user-credentials) endpoint to retrieve the issued credential data.

Reviewer: ```suggestion
The issued credential contains an `id` that you should store. You'll need this, for example, if you ever want to revoke the credential. **We recommend storing the entire credential, indexed on the credential `id`.** However, it is also an acceptable to store credential `id`s associated with a user and use the [Get Credential](/api-overview#get-credential) or [Get All User Credentials](/api-overview#get-all-user-credentials) endpoint to retrieve the full credential.
```

---

### Get Credential

> Get a previously issued <Tip type="credential"/>.

**Method**: `GET` <br />
**Path**: `/credentials/{id}`

> This endpoint returns the credential data is issued by your brand via the credential `id`. 

Reviewer: ```suggestion
This endpoint returns a credential (issued by your brand) via the credential `id`. 
```
